### ISSUE TITLE
A potential bug in TriangleMultiplicativeModule function

### ISSUE THREAD

**Issue by huangtinglin** [Graylab/GeoDock#2]
I found a potential bug in the following link:

https://github.com/Graylab/GeoDock/blob/main/geodock/model/modules/graph_module.py#L172

Shouldn't the right embedding be element-wise multiplied with the right gate? Thanks!

**Comment by Tsuchihashi-ryo** [comment]
Hello.
I also tried to use GEODOCK with NVIDIA A100 (40GB). When I ran it with two proteins (900 residues in total) I got the same error. When I ran the simulation on a smaller protein (700 residues total), the docking simulation worked. In other words, there is probably not enough memory. The total residues and memory usage are listed below, and are expected to increase approximately linearly.
200 residuals → 6GB
400 residuals → 14GB
700 residuals → 31GB
900 residuals → Error
Each one completes in 1~3 seconds, very fast and impressive!
